syntax = "proto3";

package client_replica;

import "common.proto";

service ClientReplicaGrpc {
    rpc Request (common.SignedMessage) returns (common.Empty) {}
    rpc Reply (common.Empty) returns (stream common.SignedMessage) {}
}
